x-pack/legacy/plugins/apm/server/lib/traces/get_trace_items.ts

```diff
@@ -40,22 +48,87 @@ export async function getTraceItems(
           ],
           should: {
             exists: { field: PARENT_ID }
-          }
+          },
+          must_not: { terms: { [ERROR_LOG_LEVEL]: excludedLogLevels } }
         }
       },
       sort: [
         { _score: { order: 'asc' as const } },
         { [TRANSACTION_DURATION]: { order: 'desc' as const } },
         { [SPAN_DURATION]: { order: 'desc' as const } }
       ],
-      track_total_hits: true
+      track_total_hits: true,
+      aggs: {
+        errors: {
+          filter: { term: { [PROCESSOR_EVENT]: 'error' } },
+          aggs: {
+            transactions: {
+              terms: {
+                field: TRANSACTION_ID,
+                // high cardinality
+                execution_hint: 'map' as const
+              },
+              aggs: {
+                top_transactions_errors_hits: {
+                  top_hits: { size: 100 }
+                }
+              }
+            }
+          }
+        }
+      },
+      post_filter: {
+        terms: { [PROCESSOR_EVENT]: ['span', 'transaction'] }
+      }
     }
   };
 
-  const resp = await client.search<Transaction | Span | APMError>(params);
+  const resp = await client.search(params);
+  const transactionsBuckets = resp.aggregations?.errors.transactions.buckets;
```

Comment:
I think it's actually an error bucket, isn't it?  If yes maybe naming it `errorBuckets` will be more clear.